attributes/default.rb

```diff
@@ -15,3 +15,7 @@
 
 # PGP Key Server
 default['le']['pgp_key_server'] = 'pgp.mit.edu'
+
+# Debian Release
+
+default['le']['deb'] = 'trusty'
```

Comment:
I'd like to maintain as much backward compatibility as possible. We're not going to break anything new by setting ```node['le']['deb']``` to ```node['lsb']['codename']``` as that's the current behaviour however hardcoding ```node['le']['deb']``` to "Trusty" will possibly have breaking changes.